.github/workflows/build.yml

```
@@ -232,6 +233,15 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Download artifact for arm verification
+        uses: actions/download-artifact@v2
+        if: ${{ matrix.arch == 'arm64' || matrix.arch == 'arm' }}
+        with:
+          name: ${{ env.PKG_NAME }}_${{env.version}}_linux_${{ matrix.arch }}.zip
+      - name: Unzip for ${{ matrix.arch }} linux zip
+        if: ${{ matrix.arch == 'arm64' || matrix.arch == 'arm' }}
+        run: |
+          unzip ${{ env.PKG_NAME }}_${{env.version}}_linux_${{ matrix.arch }}.zip
```

small nit: I've been putting a space between the brackets. 
```suggestion
          unzip ${{ env.PKG_NAME }}_${{ env.version }}_linux_${{ matrix.arch }}.zip
```